README.md

```diff
@@ -74,19 +74,23 @@ In this section we'll use the Azure Portal to create the required resources to h
 
 ### Configuring the Resources
 
-Once all the resources are created, you will need to get the connection information for the Postgres and Storage account to the Web App, as well as configure the resources for use.
+Once all the resources are created, you will need to get the connection information for the Postgres and Storage account to the Web App (App Service), as well as configure the resources for use.
```

Comment:
Adding this in brackets as it's called both Web App when adding it and App Service in the UI, to prevent confusion.